independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/RestClientRequestContext.java

```diff
@@ -139,9 +140,9 @@ public RestClientRequestContext(ClientImpl restClient,
         // each invocation gets a new set of properties based on the JAX-RS invoker
         this.properties = new HashMap<>(properties);
 
-        // this isn't a real configuration option because it's only really used to pass the TCKs
         disableContextualErrorMessages = Boolean
-                .parseBoolean(System.getProperty("quarkus.rest-client.disable-contextual-error-messages", "false"));
+                .parseBoolean(System.getProperty("quarkus.rest-client.disable-contextual-error-messages", "false"))
```

Comment:
Do we still need the system property check here? Since you are now reading from the Smallrye Config which already has registers `SysPropConfigSource` I would assume the extra check is not needed